### DISCUSSION TITLE
Problem showing image in PDF in production

### DISCUSSION THREAD

**Discussion by sabinnitu** [spatie/laravel-pdf#86] [Q&A]
Hello, 

I have a problem when I try to display an image in PDF only in the production server. Locally, the image works. I also have a development server _(same settings and configuration as production)_ where the image also works.

Code:
`<img src="{{ config('app.url') . "/" . $configuration->image_path }}" />`

The only difference that I saw was that in dev server the route of the image starts with http (because of the .env configs) and on production server starts with https. So I change the code to be like this:

```
@if(app()->environment('production'))
<img src="{{ "http://example.com//“ . $configuration->image_path }}" />
@else
<img src="{{ config('app.url') . "/" . $configuration->image_path }}" />
@endif
```

Same problem. How can I fix this problem in production server?
Thank you!

**Comment by RVP04** [comment]
Please check recent PR74 I think so by
